### DISCUSSION TITLE
Não consigo fazer o login, mesmo sabendo email e senha.

### DISCUSSION THREAD

**Discussion by MattSouzaBR** [community/community#28638] [Other Feature Feedback, Questions, & Ideas]
<Portuguese - Brazil> Quando efetuo o login pelo GitHub Desktop, ele solicita o login pelo navegador. Quando faço o login pelo navegador e o mesmo solicita acesso ao GitHub, eu permito mas não faz o login.

**Comment by Wenderson-Oscar** [comment]
Verifique as configurações de autenticação, ou revogue as permissões do github desktop no github e reinicie o github desktop e tente fazer login novamente.
se nada der certo reinstalar o github desktop.

**Comment by ghostinhershell** [comment]
Hi @MattSouzaBR, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
